### PR TITLE
Allow to define a minimum interest amount

### PR DESCRIPTION
**Pull request by guewen**
New feature in the module sale_payment_term_interest: a minimum interest amount
can be defined on the payment term.

### CODE REVIEW

**Comment by yvaucher** [comment]
:+1:

**Comment by nbessi** [comment]
:+1:

**Comment by mdietrichc2c** [comment]
LGTM :+1:

File: sale_payment_term_interest/i18n/fr.po [hunk 1]
```diff
@@ -42,6 +42,11 @@ msgid "Interest line"
...
 
 #. module: sale_payment_term_interest
+#: field:account.payment.term,interest_min:0
+msgid "Minimum Interest Amount"
+msgstr "Montant d'intérêt minimum"
+
+#. module: sale_payment_term_interest
 #: model:ir.model,name:sale_payment_term_interest.model_account_payment_term
 msgid "Payment Term"
 msgstr "Condition de règlement"
```
Comment (gurneyalex) [inline sale_payment_term_interest/i18n/fr.po:52]: I'd use a plural form: "Conditions" :question:

Comment (nbessi) [inline sale_payment_term_interest/i18n/fr.po:52]: In account addon the french translation is "Conditions de règlement " so I guess you are correct

**Pull request merged**